src/hooks/isCurrent.ts

```diff
@@ -8,7 +9,8 @@ import { Hook, HookContext } from '@feathersjs/feathers';
  */
 export default (): Hook => {
   return (context: HookContext): HookContext => {
-    if (context.params.user?.id !== context.id) {
+    const user = context.params.user?.permissions;
+    if ((!user.includes(UserRoles.ADMIN)) && (context.params.user?.id !== context.id)) {
```

Comment:
Please, remove not needed extra brackets

Reply:
removed cc2de6c4baad0c9ed12205e453a9abca8ed0fdf7